src/Model/Contact.php

```diff
@@ -2123,9 +2122,8 @@ public static function createFromProbe(array $user, $url, $interactive = false,
 
 		$pending = false;
 		if ($protocol == Protocol::ACTIVITYPUB) {
-			$apcontact = APContact::getByURL($ret['url'], false);
-			if (isset($apcontact['manually-approve'])) {
-				$pending = (bool)$apcontact['manually-approve'];
+			if (isset($ret['manually-approve'])) {
```

Comment:
Please simplify the conditional to a single level:
```php
if ($protocol == Protocol::ACTIVITYPUB && isset($ret['manually-approve'])) {
```

Reply:
Done!